templates/vue/src/components/SettingsModal.vue

```diff
@@ -64,8 +64,17 @@
           </b-form-group>
         </b-tab>
         <b-tab title="Advanced">
-          <b-button block variant="light" @click="exportTapestry">
-            Export Tapestry
+          <b-button
+            id="export-button"
+            block
+            variant="light"
+            :class="isExporting ? 'disabled' : ''"
+            @click="exportTapestry"
+          >
```

Comment:
I think we should also set the `disabled` prop to true when it's exporting, otherwise the button would still be clickable while it's loading (if not by the cursor, then programmatically).

Reply:
Was a little confused about this one, it's currently disabled when exporting

Reply:
I might be misunderstanding, but right now it looks to me like the class is set to disabled but the button isn't actually disabled, i.e. the `disabled` attribute of the button is not set to `true`.